src/libraries/Common/src/System/Security/Cryptography/PasswordBasedEncryption.cs

```diff
@@ -556,6 +580,12 @@ private static CryptographicException AlgorithmKdfRequiresChars(string algId)
         {
             string? algId = encryptionScheme.Algorithm;
 
+            if (!Helpers.HasSymmetricEncryption)
```

Comment:
This (+ `HasHMAC` below) are becoming something of a common pattern. Would it make sense to refactor the "check + throw an appropriate exception" logic into its own helper method?

Reply:
M...aybe?  They're here because of the UnsupportedOS flow analyzer, to show it that there's no way for Browser to make it to the annotated-as-failing Creates.  I don't know how much flexibility we have with what it understands.